statistics-dashboard-rest/src/main/java/statistics/dashboard/rest/config/ApplicationProperties.java

```diff
@@ -0,0 +1,109 @@
+package statistics.dashboard.rest.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+/**
+ * Holder for the property values from Spring injection/property loading.
+ */
+@Component
+@PropertySource({"classpath:statistics.properties"})
+public class ApplicationProperties {
+
+  // Socks proxy
+  @Value("${socks.proxy.enabled}")
+  private boolean socksProxyEnabled;
+  @Value("${socks.proxy.host}")
+  private String socksProxyHost;
+  @Value("${socks.proxy.port}")
+  private String socksProxyPort;
+  @Value("${socks.proxy.username}")
+  private String socksProxyUsername;
+  @Value("${socks.proxy.password}")
+  private String socksProxyPassword;
+
+  // Mongo
+  @Value("${mongo.hosts}")
+  private String[] mongoHosts;
+  @Value("${mongo.port}")
+  private int mongoPort;
+  @Value("${mongo.authentication.db}")
+  private String mongoAuthenticationDb;
+  @Value("${mongo.username}")
+  private String mongoUsername;
+  @Value("${mongo.password}")
+  private String mongoPassword;
+  @Value("${mongo.enable.ssl}")
+  private boolean mongoEnableSsl;
+  @Value("${mongo.application.name}")
+  private String mongoApplicationName;
+  @Value("${mongo.record.db}")
+  private String mongoRecordDb;
```

Comment:
Perhaps not call this record DB (as it does not contain records)... maybe just `mongo.database.name`? Same for the variable it sets.

Reply:
Good point, I'll change it